pyvo/registry/__init__.py

```diff
@@ -5,7 +5,9 @@
 The regtap module supports access to the IVOA Registries
 """
 
-from .regtap import search, ivoid2service, get_RegTAP_query, choose_RegTAP_service
+from .regtap import (search, ivoid2service, get_RegTAP_query,
+    choose_RegTAP_service,
+    RegistryResults, RegistryResource)
```

Comment:
Is this change intentional as part of this PR? If yes, what's the purpose?

(If the intent is to consolidate the public API and remove the `regtap` implementation detail, I would strongly suggest to do it separately from this PR.